### PR TITLE
convert some string keywords to bool

### PR DESCRIPTION
**Pull request by takumakanari**
This p-r adds new feature that converts some string keywords to boolean value.

embulk-json-path has similar feature, added at https://github.com/takumakanari/embulk-parser-json/pull/1.

### CODE REVIEW

File: src/main/java/org/embulk/parser/jsonpath/ColumnVisitorImpl.java [hunk 1]
```diff
@@ -25,6 +29,8 @@
         implements ColumnVisitor
 {
     private static final JsonParser JSON_PARSER = new JsonParser();
+    private static final List<String> BOOL_TRUE_STRINGS = ImmutableList.of("true", "1", "yes");
```
Comment (hiroyuki-sato) [inline src/main/java/org/embulk/parser/jsonpath/ColumnVisitorImpl.java]: The CSV parser treat the following value as `true`.
So I think it is better to add a some keywords (ex `On`) for true value.

How do you think?

```java
ImmutableSet.of(
        "true", "True", "TRUE",
        "yes", "Yes", "YES",
        "t", "T", "y", "Y",
        "on", "On", "ON",
        "1");
```

https://github.com/embulk/embulk/blob/master/embulk-standards/src/main/java/org/embulk/standards/CsvParserPlugin.java#L36-L41

**Review by hiroyuki-sato — COMMENTED** [review]
Sorry, I forgot to push `submit review` button.
I think it is better to add some strings like the below for compatibility CSV parser plugin behavior. 

How do you think?

```java
private static final List<String> BOOL_TRUE_STRINGS = ImmutableList.of("true", "1", "yes","y","t","on");
 ```

File: src/main/java/org/embulk/parser/jsonpath/ColumnVisitorImpl.java (hunk 1 above)
Comment (takumakanari) [reply src/main/java/org/embulk/parser/jsonpath/ColumnVisitorImpl.java]: - Keywords will be compared with case-ignored, so we don't need all case variations
  https://github.com/hiroyuki-sato/embulk-parser-jsonpath/pull/13/files#diff-2d0fb975204a675b52cc8ea532734971R90

- Adding more words (e.g. 'on', 'y') is good idea. I'll handle it.

Comment (takumakanari) [reply src/main/java/org/embulk/parser/jsonpath/ColumnVisitorImpl.java]: BTW: Is it better to follow CSVParserPlugin to convert string keywords to bool? What do you think about this?

**Comment by takumakanari** [comment]
@hiroyuki-sato Thanks for your review. As we discussed on Twitter, I added more words to convert to bool.
Could you check it again?

**Comment by hiroyuki-sato** [comment]
Thank you!
LGTM 👍
I've merged!
(We discussed this topic on [twitter](https://twitter.com/hiroysato/status/841143199250837504)) (Japanese text)

**Comment by takumakanari** [comment]
Thank you!

**Pull request merged**